examples/README.md

```diff
@@ -28,7 +28,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-slurm-sharedvpc.yaml](#hpc-slurm-sharedvpcyaml-) ![community-badge]
   * [client-google-cloud-storage.yaml](#client-google-cloud-storageyaml--) ![community-badge] ![experimental-badge]
   * [hpc-slurm-gromacs.yaml](#hpc-slurm-gromacsyaml--) ![community-badge] ![experimental-badge]
-  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge]
+  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge] ![deprecated-badge]
```

Comment:
nit: I would move it to the bottom of the list. That is where we previously had the deprecated v4 examples.

Reply:
Done